src/GovUk.Education.ExploreEducationStatistics.Admin/Security/AuthorizationHandlers/UpdateSpecificMethodologyAuthorizationHandler.cs

```diff
@@ -13,33 +19,105 @@ public class UpdateSpecificMethodologyRequirement : IAuthorizationRequirement
     public class UpdateSpecificMethodologyAuthorizationHandler : 
         AuthorizationHandler<UpdateSpecificMethodologyRequirement, Methodology>
     {
+        private readonly ContentDbContext _contentDbContext;
         private readonly IMethodologyRepository _methodologyRepository;
+        private readonly IPublicationRepository _publicationRepository;
+        private readonly IUserPublicationRoleRepository _userPublicationRoleRepository;
+        private readonly IUserReleaseRoleRepository _userReleaseRoleRepository;
 
-        public UpdateSpecificMethodologyAuthorizationHandler(IMethodologyRepository methodologyRepository)
+        public UpdateSpecificMethodologyAuthorizationHandler(
+            ContentDbContext contentDbContext,
+            IMethodologyRepository methodologyRepository, 
+            IPublicationRepository publicationRepository,
+            IUserPublicationRoleRepository userPublicationRoleRepository, 
+            IUserReleaseRoleRepository userReleaseRoleRepository)
         {
+            _contentDbContext = contentDbContext;
             _methodologyRepository = methodologyRepository;
+            _userPublicationRoleRepository = userPublicationRoleRepository;
+            _userReleaseRoleRepository = userReleaseRoleRepository;
+            _publicationRepository = publicationRepository;
         }
 
         protected override async Task HandleRequirementAsync(AuthorizationHandlerContext context,
             UpdateSpecificMethodologyRequirement requirement,
             Methodology methodology)
         {
+            // An Approved Methodology cannot be updated.  Instead, it should firstly be unapproved if permissions
+            // allow and then updated.
+            if (methodology.Approved)
```

Comment:
This breaks the list of unpublished releases used on the Sign Off tab which is returned by method GetUnpublishedReleasesUsingMethodology currently secured by this handler, when the methodology is already approved. If a user who can approve/unapprove goes back to the Sign Off tab they get an error.

The GetUnpublishedReleasesUsingMethodology  needs changing to be secured by those who can change status, rather than approve.

Reply:
This is fixed by #2769, by changing how `GetUnpublishedReleasesUsingMethodology ` is secured.